hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestSnapshotCache.java

```diff
@@ -82,7 +82,7 @@ static void beforeAll() throws Exception {
   void setUp() {
     // Reset cache for each test case
     snapshotCache = new SnapshotCache(
-        omSnapshotManager, cacheLoader, CACHE_SIZE_LIMIT);
+        omSnapshotManager, cacheLoader, CACHE_SIZE_LIMIT, 0);
```

Comment:
How setting the clean up interval to 0 is working for the test? I don't think it is correct to set clean-up to 0 or tests are more correct.

Reply:
We don't want the cleanup service to run here, since everything is mocked here.